scripts/lua/header.lp

```diff
@@ -16,8 +16,8 @@ theme = pihole.webtheme()
 -- Get name of script by matching whatever is after the last "/" in the URI
 scriptname = mg.script_name:match(".*/(.*).lp$"):gsub("-", "/")
 
--- Fall back to "index.lp" if no match is found (e.g. when accessing the root)
-if scriptname == nil or string.len(scriptname) == 0 then scriptname = "index.lp" end
+-- Fall back to "index" if no match is found (e.g. when accessing the root)
+if scriptname == nil or string.len(scriptname) == 0 then scriptname = "index" end
```

Comment:
Makes me wonder if this should be "." like in login.js

https://github.com/pi-hole/web/blob/cc1cc2859ecb543e3cf7f1874e8a74ef0bd49b8a/scripts/js/login.js#L14-L26

Reply:
I guess there is no difference between `location.replace(".");` and `location.replace("");`